### PR TITLE
[Feature][Docs] Add English version of "system-manual" to the documentation

### PR DESCRIPTION
**Pull request by Segun-Ogundipe**
# What is the purpose of the pull request

This adds the English version of the "system-manual" to the user documentation.

# Change Log

- Add "docs/en-us/1.3.2/user-doc/system-manual.md"
- Add "img/create_project_en.png"

# How to verify this PR

- Clone the branch
- npm i
- docsite start
- Go to "localhost:8080/en-us/docs/1.3.2/user_doc/system-manual.html"

### CODE REVIEW

**Comment by lgcareer** [comment]
@Segun-Ogundipe 
Thanks for your contribution.
I saw the spark task node page still used chinese.Here need change to the english.

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 1]
```diff
@@ -0,0 +1,883 @@
...
+  </p>
+
+#### 2.2 Project home
+
+- Click the project name link on the project management page to enter the project home page, as shown in the figure below, the project home page contains the task status statistics, process status statistics, and workflow definition statistics of the project.
+  <p align="center">
+     <img src="/img/project_home_en.png" width="80%" />
+  </p>
+
+- Task status statistics: within the specified time range, count the number of tasks in the task instance as successful submission, running, ready to pause, pause, ready to stop, stop, failure, success, fault tolerance, kill, and waiting threads
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:34]: Please keep task status count translate the same with work flow count translate.

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 2]
```diff
@@ -0,0 +1,883 @@
...
+- Process status statistics: within the specified time range, count the number of the status of the workflow instance as submission success, running, ready to pause, pause, ready to stop, stop, failure, success, fault tolerance, kill, and waiting threads
+- Workflow definition statistics: Count the workflow definitions created by users and the workflow definitions granted to this user by the administrator
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:36]: created by this user

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 3]
```diff
@@ -0,0 +1,883 @@
...
+
+#### 2.3 Workflow definition
+
+#### <span id=creatDag>2.3.1 Create a workflow definition</span>
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:40]: Create workflow definition

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 4]
```diff
@@ -0,0 +1,883 @@
...
+
+- Click Project Management -> Workflow -> Workflow Definition to enter the workflow definition page, and click the "Create Workflow" button to enter the **workflow DAG edit** page, as shown in the following figure:
+  <p align="center">
+      <img src="/img/dag5.png" width="80%" />
+  </p>
+- Drag in the toolbar <img src="/img/shell.png" width="35"/> Add a Shell task to the drawing board, as shown in the figure below:
+  <p align="center">
+      <img src="/img/shell-en.png" width="80%" />
+  </p>
+- **Add parameter settings for shell tasks:**
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:50]: for this shell task

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 5]
```diff
@@ -0,0 +1,883 @@
...
+
+1. Fill in the "Node Name", "Description", and "Script" fields;
+2. Check “Normal” for “Run Flag”. If “Prohibit Execution” is checked, the task will not be executed when running the workflow;
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:53]: when run the workflow

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 6]
```diff
@@ -0,0 +1,883 @@
...
+3. Select "Task Priority": When the number of worker threads is insufficient, high-level tasks will be executed first in the execution queue, and tasks with the same priority will be executed in the order of first in, first out;
+4. Timeout alarm (not required): Check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will be sent and the task timeout fails;
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:55]: not required to optional

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 7]
```diff
@@ -0,0 +1,883 @@
...
+5. Resources (optional). Resource files are files created or uploaded on the Resource Center -> File Management page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`;
+6. Custom parameters (not required), refer to [Custom Parameters](#UserDefinedParameters);
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:57]: not required to optional

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 8]
```diff
@@ -0,0 +1,883 @@
...
+7. Click the "Confirm Add" button to save the task settings.
+
+- **Increase the order of task execution:** Click the icon in the upper right corner <img src="/img/line.png" width="35"/> to connect the task; as shown in the figure below, task 2 and task 3 are executed in parallel, When task 1 is executed, tasks 2 and 3 will be executed simultaneously.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:60]: When task 1 finish execute

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 9]
```diff
@@ -0,0 +1,883 @@
...
+
+Click Project Management -> Workflow -> Workflow Definition to enter the workflow definition page, as shown below:
+
+<p align="center">
+<img src="/img/work_list_en.png" width="80%" />
+</p>
+The operation functions of the workflow definition list are as follows:
+
+- **Edit:** Only "offline" workflow definitions can be edited. Workflow DAG editing is the same as [Create Workflow Definition](#creatDag).
+- **Online:** When the workflow status is "Offline", the workflow is online. Only the workflow in the "Online" state can run, but cannot be edited.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:87]: When the workflow status is "Offline", used to online workflow.

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 10]
```diff
@@ -0,0 +1,883 @@
...
+- **Offline:** When the workflow status is "Online", the workflow is offline. Only the workflow in the "Offline" state can be edited, but not run.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:88]: When the workflow status is "Online", used to offline workflow.

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 11]
```diff
@@ -0,0 +1,883 @@
...
+- **Delete:** Delete the workflow definition.
+- **Download:** Download workflow definition to local.
+- **Tree Diagram:** Display the task node type and task status in a tree structure, as shown in the figure below:
+  <p align="center">
+      <img src="/img/tree_en.png" width="80%" />
+  </p>
+
+#### <span id=runWorkflow>2.3.3 Run the workflow</span>
+
+- Click Project Management -> Workflow -> Workflow Definition to enter the workflow definition page, as shown in the figure below, click the "Go Online" button <img src="/img/online.png" width="35"/>，Go online workflow.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:101]: Go online workflow -> to online workflow

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 12]
```diff
@@ -0,0 +1,883 @@
...
+  </p>
+
+- Click the "Run" button to pop up the startup parameter setting pop-up box, as shown in the figure below, set the startup parameters, click the "Run" button in the pop-up box, the workflow starts running, and the workflow instance page generates a workflow instance.
+     <p align="center">
+       <img src="/img/run_work_en.png" width="80%" />
+     </p>  
+  <span id=runParamers>Description of workflow operating parameters:</span> 
+       
+      * Failure strategy: When a task node fails to execute, other parallel task nodes need to execute the strategy. "Continue" means: after a certain task fails, other task nodes execute normally; "End" means: terminate all tasks being executed, and terminate the entire process.
+      * Notification strategy: When the process is over, the process execution information notification email is sent according to the process status, including any status is not sent, successful, failed, successful or failed.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:113]:
```suggestion
      * Notification strategy: When the process is over, the process execution information notification email is sent according to the process status, _including any status is not sent, successful, failed, successful or failed._ need improve
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 13]
```diff
@@ -0,0 +1,883 @@
...
+      * Recipient: Select notification policy||Timeout alarm||When fault tolerance occurs, process information or alarm email will be sent to the recipient list.
+      * Cc: Select the notification strategy||Timeout alarm||When fault tolerance occurs, the process information or warning email will be copied to the CC list.
+      * Complement: Two modes including serial complement and parallel complement. Serial complement: within the specified time range, the complement is executed sequentially from the start date to the end date, and only one process instance is generated; parallel complement: within the specified time range, multiple days are complemented at the same time to generate N process instances.
+    * Complement: Execute the workflow definition of the specified date, you can select the time range of the complement (currently only supports the complement for consecutive days), for example, you need to supplement the data from September 19 to October 16, as shown in the following figure:
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:120]:
```suggestion
    * supplement the data from September 19 to October 16 
```date not the same as cn version

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 14]
```diff
@@ -0,0 +1,883 @@
...
+- Create timing: Click Project Management->Workflow->Workflow Definition, enter the workflow definition page, go online the workflow, click the "timing" button <img src="/img/timing.png" width="35"/> ,The timing parameter setting dialog box pops up, as shown in the figure below:
+  <p align="center">
+      <img src="/img/time_schedule_en.png" width="80%" />
+  </p>
+- Choose the start and end time. In the start and end time range, the workflow is run at regular intervals; not in the start and end time range, no more regular workflow instances are generated.
+- Add a timing that is executed once every day at 5 AM, as shown in the following figure:
+  <p align="center">
+      <img src="/img/timer-en.png" width="80%" />
+  </p>
+- Failure strategy, notification strategy, process priority, worker grouping, notification group, recipient, and CC are the same as [workflow running parameters](#runParamers).
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:140]:
```suggestion
- worker grouping
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 15]
```diff
@@ -0,0 +1,883 @@
...
+- Click the "Create" button to create the timing successfully. At this time, the timing status is "**Offline**" and the timing needs to be **Online** to take effect.
+- Timed online: Click the "timing management" button <img src="/img/timeManagement.png" width="35"/>, enter the timing management page, click the "online" button, the timing status will change to "online", as follows As shown in the figure, the workflow takes effect regularly.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:142]:
```suggestion
- Timing online: Click the "timing management" button <img src="/img/timeManagement.png" width="35"/>, enter the timing management page, click the "online" button, the timing status will change to "online", as shown in the below figure, the workflow takes effect regularly.
```

File: docs/en-us/1.3.2/user_doc/system-manual.md (hunk 7 above)
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:57]:
```suggestion
6. Custom parameters (optional), refer to [Custom Parameters](#UserDefinedParameters);
```

Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:57]:
```suggestion
6. Custom parameters (optional), refer to [Custom Parameters](#UserDefinedParameters);
```

File: docs/en-us/1.3.2/user_doc/system-manual.md (hunk 6 above)
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:55]:
```suggestion
4. Timeout alarm (optional): Check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will be sent and the task timeout fails;
```

**Review by AhahaGe — COMMENTED** [review]
Review to 2.3.5 import workflow

**Comment by AhahaGe** [comment]
I just reviewed to 2.3.5, any one review and continue from 2.3.5.

**Comment by Segun-Ogundipe** [comment]
Hi, @AhahaGe , @dailidong, I've made the requested changes kindly check

**Comment by AhahaGe** [comment]
Hi, I suggest we translate in the same format, content of task instance part is the same as workflow part but translate different, all be optional, or all be not required, not part optional ,part not required. Example in en version is supplement the data from September 19 to October 16,  but may 1 to 10 in cn version. @Segun-Ogundipe @dailidong

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 16]
```diff
@@ -0,0 +1,876 @@
...
+- Timing online: Click the "timing management" button <img src="/img/timeManagement.png" width="35"/>, enter the timing management page, click the "online" button, the timing status will change to "online", as shown in the below figure, the workflow takes effect regularly.
+  <p align="center">
+      <img src="/img/time-manage-list-en.png" width="80%" />
+  </p>
+
+#### 2.3.5 Import workflow
+
+Click Project Management -> Workflow -> Workflow Definition to enter the workflow definition page, click the "Import Workflow" button to import the local workflow file, the workflow definition list displays the imported workflow, and the status is offline.
+
+#### 2.4 Workflow example
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:153]:
```suggestion
#### 2.4 Workflow instance
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 17]
```diff
@@ -0,0 +1,876 @@
...
+
+#### 2.4.1 View workflow example
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:155]:
```suggestion
#### 2.4.1 View workflow instance
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 18]
```diff
@@ -0,0 +1,876 @@
...
+- Click Project Management -> Workflow -> Workflow Instance to enter the workflow instance page, and click the workflow name to enter the workflow DAG page;
+- Double-click the task node, as shown in the figure below, click "View History" to jump to the task instance page, and display a list of task instances running by the workflow instance
+   <p align="center">
+     <img src="/img/task_history_en.png" width="80%" />
+   </p>
+
+#### 2.4.4 View operating parameters
+
+- Click Project Management -> Workflow -> Workflow Instance to enter the workflow instance page, and click the workflow name to enter the workflow DAG page;
+- Click the icon in the upper left corner<img src="/img/run_params_button.png" width="35"/>，View the startup parameters of the workflow instance; click the icon<img src="/img/global_param.png" width="35"/>，View the global and local parameters of the workflow instance, as shown in the following figure:
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:188]:
```suggestion
- Click the icon in the upper left corner<img src="/img/run_params_button.png" width="35"/>，View the startup parameters of the workflow instance; click the icon<img src="/img/global_param.png" width="35"/>, View the global and local parameters of the workflow instance, as shown in the following figure:
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 19]
```diff
@@ -0,0 +1,876 @@
...
+
+#### 2.4.4 Workflow instance operation function
+
+Click Project Management -> Workflow -> Workflow Instance to enter the Workflow Instance page, as shown in the figure below:
+
+  <p align="center">
+    <img src="/img/instance-list-en.png" width="80%" />
+  </p>
+
+- **Edit：** Only terminated processes can be edited. Click the "Edit" button or the name of the workflow instance to enter the DAG editing page. After editing, click the "Save" button to pop up the Save DAG pop-up box, as shown in the figure below. In the pop-up box, check "Whether to update to workflow definition" and save After that, the workflow definition will be updated; if it is not checked, the workflow definition will not be updated.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:201]:
```suggestion
- **Edit：** Only terminated processes can be edited. Click the "Edit" button or the name of the workflow instance to enter the DAG edit page. After edit, click the "Save" button to pop up the Save DAG pop-up box, as shown in the figure below. In the pop-up box, check "Whether to update to workflow definition" and save After that, the workflow definition will be updated; if it is not checked, the workflow definition will not be updated.
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 20]
```diff
@@ -0,0 +1,876 @@
...
+     <p align="center">
+       <img src="/img/editDag-en.png" width="80%" />
+     </p>
+- **Rerun：** Re-execute the terminated process.
+- **Recovery failed：** For failed processes, you can perform recovery operations, starting from the failed node.
+- **stop：** To **stop** the running process, the background will first `kill`worker process, and then execute `kill -9` operation
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:207]:
```suggestion
- **stop：** To **stop** the running process, the background will first `kill`worker process, and then execute `kill -9` operation       just kill process, why we need to kill worker process?
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 21]
```diff
@@ -0,0 +1,876 @@
...
+
+### 3. Resource Center
+
+#### 3.1 hdfs resource configuration
+
+- Upload resource files and udf functions, all uploaded files and resources will be stored on hdfs, so the following configuration items are required:
+
+```
+
+conf/common/common.properties # Users who have permission to create directories under the HDFS root path
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:236]:
```suggestion
conf/common/common.properties 
# Users who have permission to create directories under the HDFS root path
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 22]
```diff
@@ -0,0 +1,876 @@
...
+hdfs.root.user=hdfs # data base dir, resource file will store to this hadoop hdfs path, self configuration, please make sure the directory exists on hdfs and have read write permissions。"/escheduler" is recommended
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:237]:
```suggestion
hdfs.root.user=hdfs 
# data base dir, resource file will store to this hadoop hdfs path, self configuration, please make sure the directory exists on hdfs and have read write permissions。"/escheduler" is recommended
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 23]
```diff
@@ -0,0 +1,876 @@
...
+data.store2hdfs.basepath=/dolphinscheduler # resource upload startup type : HDFS,S3,NONE
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:238]:
```suggestion
data.store2hdfs.basepath=/dolphinscheduler 
# resource upload startup type : HDFS,S3,NONE
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 24]
```diff
@@ -0,0 +1,876 @@
...
+res.upload.startup.type=HDFS # whether kerberos starts
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:239]:
```suggestion
res.upload.startup.type=HDFS
# whether kerberos starts
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 25]
```diff
@@ -0,0 +1,876 @@
...
+hadoop.security.authentication.startup.state=false # java.security.krb5.conf path
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:240]:
```suggestion
hadoop.security.authentication.startup.state=false 
# java.security.krb5.conf path
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 26]
```diff
@@ -0,0 +1,876 @@
...
+java.security.krb5.conf.path=/opt/krb5.conf # loginUserFromKeytab user
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:241]:
```suggestion
java.security.krb5.conf.path=/opt/krb5.conf 
# loginUserFromKeytab user
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 27]
```diff
@@ -0,0 +1,876 @@
...
+login.user.keytab.username=hdfs-mycluster@ESZ.COM # loginUserFromKeytab path
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:242]:
```suggestion
login.user.keytab.username=hdfs-mycluster@ESZ.COM 
# loginUserFromKeytab path
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 28]
```diff
@@ -0,0 +1,876 @@
...
+login.user.keytab.path=/opt/hdfs.headless.keytab
+
+conf/common/hadoop.properties # ha or single namenode,If namenode ha needs to copy core-site.xml and hdfs-site.xml # to the conf directory，support s3，for example : s3a://dolphinscheduler
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:245]:
```suggestion
conf/common/hadoop.properties 
# ha or single namenode,If namenode ha needs to copy core-site.xml and hdfs-site.xml # to the conf directory，support s3，for example : s3a://dolphinscheduler
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 29]
```diff
@@ -0,0 +1,876 @@
...
+fs.defaultFS=hdfs://mycluster:8020
+#resourcemanager ha note this need ips , this empty if single
+yarn.resourcemanager.ha.rm.ids=192.168.xx.xx,192.168.xx.xx # If it is a single resourcemanager, you only need to configure one host name. If it is resourcemanager HA, the default configuration is fine
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:248]:
```suggestion
yarn.resourcemanager.ha.rm.ids=192.168.xx.xx,192.168.xx.xx 
# If it is a single resourcemanager, you only need to configure one host name. If it is resourcemanager HA, the default configuration is fine
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 30]
```diff
@@ -0,0 +1,876 @@
...
+yarn.application.status.address=http://xxxx:8088/ws/v1/cluster/apps/%s
+
+````
+
+- Only one address needs to be configured for yarn.resourcemanager.ha.rm.ids and yarn.application.status.address, and the other address is empty.
+- You need to copy core-site.xml and hdfs-site.xml from the conf directory of the Hadoop cluster to the conf directory of the dolphinscheduler project, and restart the api-server service.
+
+#### 3.2 File management
+
+> It is the management of various resource files, including creating basic txt/log/sh/conf/py/java and other files, uploading jar packages and other types of files, which can be edited, renamed, downloaded, deleted and other operations.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:258]:
```suggestion
> It is the management of various resource files, including creating basic txt/log/sh/conf/py/java and other files, uploading jar packages and other types of files, and can do edit, rename, download, delete and other operations.
```

**Review by AhahaGe — COMMENTED** [review]
I've reviewed to 5. Security Center

**Comment by AhahaGe** [comment]
@Segun-Ogundipe  If u think need not to edit, please feel free to comment out.

**Comment by Segun-Ogundipe** [comment]
> @Segun-Ogundipe If u think need not to edit, please feel free to comment out.

Hi @AhahaGe, the edits make it clearer

**Comment by Segun-Ogundipe** [comment]
Hi @AhahaGe, I have updated the PR with the changes you suggested. Please check

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 31]
```diff
@@ -0,0 +1,886 @@
...
+- IP/Hostname: Enter the IP connected to Spark
+- Port: Enter the port connected to Spark
+- Username: Set the username for connecting to Spark
+- Password: Set the password for connecting to Spark
+- Database name: Enter the name of the database connected to Spark
+- Jdbc connection parameters: parameter settings for Spark connection, filled in in JSON form
+
+### 5. Security Center (Permission System)
+
+     * Only the administrator account in the security center has the authority to operate. It has functions such as queue management, tenant management, user management, alarm group management, worker group management, token management, etc. In the user management module, resources, data sources, projects, etc. Authorization
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:422]:
```suggestion
     * Only the administrator account has the authority to operate security center. It has functions such as queue management, tenant management, user management, alarm group management, worker group management, token management etc. We can authorize resources, data sources, projects in the user management module.
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 32]
```diff
@@ -0,0 +1,886 @@
...
+   <img src="/img/mr_edit_en.png" width="80%" />
+ </p>
+
+- Program type: select Python language
+- Main jar package: is the Python jar package for running MR
+- Other parameters: support -D, -mapper, -reducer, -input -output format, here you can set the input of user-defined parameters, such as:
+- -mapper "mapper.py 1" -file mapper.py -reducer reducer.py -file reducer.py –input /journey/words.txt -output /journey/out/mr/\${currentTimeMillis}
+- The mapper.py 1 after -mapper is two parameters, the first parameter is mapper.py, and the second parameter is 1
+- Resource: If the resource file is referenced in other parameters, you need to select and specify in the resource
+- User-defined parameter: It is a user-defined parameter of the MR part, which will replace the content with \${variable} in the script
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:750]:
```suggestion
- User-defined parameter: It is a local user-defined parameter of the MR, which will replace the content with \${variable} in the script
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 33]
```diff
@@ -0,0 +1,886 @@
...
+   <img src="/img/mr_java_en.png" width="80%" />
+ </p>
+
+- The class of the main function: is the full path of the Main Class, the entry point of the MR program
+- Program type: select JAVA language
+- Main jar package: is the MR jar package
+- Command line parameters: set the input parameters of the MR program and support the substitution of custom parameter variables
+- Other parameters: support -D, -files, -libjars, -archives format
+- Resource: If the resource file is referenced in other parameters, you need to select and specify in the resource
+- User-defined parameter: It is a user-defined parameter of the MR part, which will replace the content with \${variable} in the script
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:736]:
```suggestion
- User-defined parameter: It is a local user-defined parameter of the MR, which will replace the content with \${variable} in the script
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 34]
```diff
@@ -0,0 +1,886 @@
...
+   <img src="/img/python-en.png" width="80%" />
+ </p>
+
+- Script: Python program developed by the user
+- Resources: refers to the list of resource files that need to be called in the script
+- User-defined parameter: It is a local user-defined parameter of Python, which will replace the content with \${variable} in the script
+
+#### 7.9 Flink Node
+
+- Drag in the toolbar<img src="/img/flink.png" width="35"/>The task node to the drawing board, as shown in the following figure:
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:768]:
```suggestion
- Drag in the toolbar<img src="/img/flink.png" width="35"/> task node to the drawing board, as shown in the following figure:
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 35]
```diff
@@ -0,0 +1,886 @@
...
+- Command line parameters: Set the input parameters of the Spark program and support the substitution of custom parameter variables.
+- Other parameters: support --jars, --files, --archives, --conf format
+- Resource: If the resource file is referenced in other parameters, you need to select and specify in the resource
+- Custom parameter: It is a local user-defined parameter of Flink, which will replace the content with \${variable} in the script
+
+Note: JAVA and Scala are only used for identification, there is no difference, if it is Flink developed by Python, there is no class of the main function, the others are the same
+
+#### 7.10 http Node
+
+- Drag in the toolbar<img src="/img/http.png" width="35"/>The task node to the drawing board, as shown in the following figure:
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:791]:
```suggestion
- Drag in the toolbar<img src="/img/http.png" width="35"/> task node to the drawing board, as shown in the following figure:
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 36]
```diff
@@ -0,0 +1,886 @@
...
+- Node name: The node name in a workflow definition is unique.
+- Run flag: Identifies whether this node can be scheduled normally, if it does not need to be executed, you can turn on the prohibition switch.
+- Descriptive information: describe the function of the node.
+- Task priority: When the number of worker threads is insufficient, they are executed in order from high to low, and when the priority is the same, they are executed according to the first-in first-out principle.
+- Worker grouping: Tasks are assigned to the machines of the worker group to execute. If Default is selected, a worker machine will be randomly selected for execution.
+- Number of failed retry attempts: The number of times the task failed to be resubmitted. It supports drop-down and hand-filling.
+- Failed retry interval: The time interval for resubmitting the task after a failed task. It supports drop-down and hand-filling.
+- Timeout alarm: Check the timeout alarm and timeout failure. When the task exceeds the "timeout period", an alarm email will be sent and the task execution will fail.
+- Request address: http request URL.
+- Request type: support GET, POSt, HEAD, PUT, DELETE.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:806]:
```suggestion
- Request type: support GET, POST, HEAD, PUT, DELETE.
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 37]
```diff
@@ -0,0 +1,886 @@
...
+- Request parameters: Support Parameter, Body, Headers.
+- Verification conditions: support default response code, custom response code, content included, content not included.
+- Verification content: When the verification condition selects a custom response code, the content contains, and the content does not contain, the verification content is required.
+- Custom parameter: It is a user-defined parameter of http part, which will replace the content with \${variable} in the script.
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:810]:
```suggestion
- Custom parameter: It is a local user-defined parameter of http, which will replace the content with \${variable} in the script.
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 38]
```diff
@@ -0,0 +1,886 @@
...
+- Drag in the toolbar<img src="/img/datax.png" width="35"/>Task node into the drawing board
+
+  <p align="center">
+   <img src="/img/datax-en.png" width="80%" />
+  </p>
+
+- Custom template: When you turn on the custom template switch, you can customize the content of the json configuration file of the datax node (applicable when the control configuration does not meet the requirements)
+- Data source: select the data source to extract the data
+- sql statement: the sql statement used to extract data from the target database, the sql query column name is automatically parsed when the node is executed, and mapped to the target table synchronization column name. When the source table and target table column names are inconsistent, they can be converted by column alias (as)
+- Target library: select the target library for data synchronization
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:823]:
```suggestion
- Target database: select the target database for data synchronization
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 39]
```diff
@@ -0,0 +1,886 @@
...
+- Target table: the name of the target table for data synchronization
+- Pre-sql: Pre-sql is executed before the sql statement (executed by the target library).
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:825]:
```suggestion
- Pre-sql: Pre-sql is executed before the sql statement (executed by the target database).
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 40]
```diff
@@ -0,0 +1,886 @@
...
+- Post-sql: Post-sql is executed after the sql statement (executed by the target library).
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:826]:
```suggestion
- Post-sql: Post-sql is executed after the sql statement (executed by the target database).
```

File: docs/en-us/1.3.2/user_doc/system-manual.md [hunk 41]
```diff
@@ -0,0 +1,886 @@
...
+      * N days before：$[yyyyMMdd-N]
+      * Next N hours：$[HHmmss+N/24]
+      * First N hours：$[HHmmss-N/24]
+      * Next N minutes：$[HHmmss+N/24/60]
+      * First N minutes：$[HHmmss-N/24/60]
+
+#### 8.3 <span id=UserDefinedParameters>User-defined parameters</span>
+
+- User-defined parameters are divided into global parameters and local parameters. Global parameters are global parameters passed when saving workflow definitions and workflow instances. Global parameters can be referenced in the local parameters of any task node in the entire process.
+  example：
```
Comment (AhahaGe) [inline docs/en-us/1.3.2/user_doc/system-manual.md:874]:
```suggestion
  For example：
```

**Review by AhahaGe — COMMENTED** [review]
Finished review.

**Comment by dailidong** [comment]
+1
good job

**Comment by dailidong** [comment]
Thanks for your first contribution, looking forward for your deep participation, if you meet any problem, please send mail to dev@dolphinscheduler.apache.org,  many guys are very enthusiastic and willing to help

**Comment by dailidong** [comment]
when github action work done, this document will be shown on official website

**Comment by CalvinKirs** [comment]
Hi, @Segun-Ogundipe ,I noticed that you did not add a link to this document, which will cause it to not be displayed correctly on our official website. You need to modify docs.json and other files. Can you submit another PR to complete this work?

**Comment by Segun-Ogundipe** [comment]
Hi, @CalvinKirs, the link was already added to site_config/docs1.3.2.js. That's why I didn't edit the file. I checked the link on the official website after the merge and it's working. Please let me know if am missing anything.

**Pull request merged**